src/marketplace-custom-config.js

```diff
@@ -1,77 +1,150 @@
 /*
  * Marketplace specific configuration.
+ *
+ * Every filter needs to have following keys:
+ * - **id**: unique id of the filter
+ * - **label**: The default label of the filter
+ * - **type**: string that represents one of the existing filter components
+ * - **group**: is this 'primary' or 'secondary' filter?
```

Comment:
There are only two groups. Could this default to one, and be a boolean. E.g. by default filters are secondary, and the option to make them primary is `isPrimary` in the config.

It would also be really good to briefly explain here what primary and secondary mean in the UI.

Reply:
Good point about opening the concept a bit more! 

However, I wouldn't turn this to a flag. That would restrict possible customizations. We currently have 3 rendering containers for filters - it could also be more (e.g. top of results, side of results + hidden panel). It's also a good place to check the options (read about 'secondary' => type search 'secondary' from codebase).